lib/graphql-anycable.rb

```diff
@@ -6,6 +6,7 @@
 require_relative "graphql/anycable/cleaner"
 require_relative "graphql/anycable/config"
 require_relative "graphql/anycable/railtie" if defined?(Rails)
+require_relative "graphql/adapters/base_job"
```

Comment:
Move into `graphql/anycable/railtie` and load only if ActiveJob is present (using `ActiveSupport.on_load(:active_job) { ... }` hook).

Reply:
yeah, I done it, but I had an error in this case

```
initializer 'graphql_anycable.load_trigger_job' do
  ActiveSupport.on_load(:active_job) do
    require "graphql/adapters/trigger_job"
  end
end
```

in that case, my application (rails 6, with configurated active_job) does not load it, because my application does not call any `ActiveJob` jobs in the code

I checked and saw, that `ActiveSupport.on_load(:active_job)` loads only when my rails application calls any jobs

but, maybe I misunderstood something.

Currently, I just commented `ActiveSupport.on_load(:active_job)` and I think, if we are in `Railtie`, we can `require` `TriggerJob` in `initialize` block

Reply:
thanks for that!

The trouble was with my test application